python/ml4ir/applications/ranking/config/keys.py

```diff
@@ -9,6 +9,9 @@ class LossKey(Key):
     RANK_ONE_LISTNET = "rank_one_listnet"
     SOFTMAX_CROSS_ENTROPY = "softmax_cross_entropy"
 
+    AUX_ONE_HOT_CROSS_ENTROPY = "aux_one_hot_cross_entropy"
```

Comment:
@mohazahran I've renamed the losses, but at a high level, they are the same.

The one hot aux loss converts `y_true` to one hot labels and the softmax loss converts them to softmax probabilities.